pyscf/adc/radc.py

```diff
@@ -94,47 +99,70 @@ def compute_amplitudes(myadc, eris):
 
     eris_oooo = eris.oooo
     eris_ovoo = eris.ovoo
-    eris_ovov = eris.ovov
     eris_oovv = eris.oovv
     eris_ovvo = eris.ovvo
-    eris_ovvv = eris.ovvv
 
-    e = myadc.mo_energy
+    # Compute first-order doubles t2 (tijab)
 
-    d_ij = e[:nocc][:,None] + e[:nocc]
+    v2e_oovv = eris_ovvo[:].transpose(0,3,1,2).copy()
```

Comment:
Dumb question: this is conversion from chemists' to physicists' notation?

Reply:
Does spliced `transpose` not already return a copy?

Reply:
Yes, this is a conversion from chemists' to physicists' notation. Also, sliced transpose does not return a copy.